e2e/cleanup_test.go

```diff
@@ -53,8 +53,10 @@ func testCleanup() {
 						topolvmFinalize = true
 					}
 				}
+				// Even if node finalize is skipped, the finalizer is still present on the node
+				// The finalizer is added by the metrics-exporter runner from topolvm-node
```

Comment:
I was surprised to find out that topolvm-node's metrics-exporter is responsible for patching Node finalizers, because I thought it would only do metrics. 

[Link to code](https://github.com/topolvm/topolvm/blob/9eea3dd64b3123a378c86c6bdeb485e9fdbba537/runners/metrics_exporter.go#L131-L150)

Perhaps it would be good to refactor so that patching the Node is the responsibility of a different function in topolvm-node? (Not in this PR, I will raise a separate issue and PR if there's agreement. :slightly_smiling_face: )